src/NzbDrone.Api/Series/SeriesModule.cs

```diff
@@ -109,7 +110,12 @@ private List<SeriesResource> AllSeries()
 
         private int AddSeries(SeriesResource seriesResource)
         {
-            return GetNewId<Core.Tv.Series>(_seriesService.AddSeries, seriesResource);
+            var options = new AddSeriesOptions();
+            options.InjectFrom(Request.Body);
```

Comment:
Won't SeriesResource and AddSeriesOptions both come from the same object? Probably better to add a property 'addOptions'.